### DISCUSSION TITLE
Not approved for account after two days and now won't let me sign up or sign in

### DISCUSSION THREAD

**Discussion by HollywoodPizza** [mastodon/mastodon#21238] [Q&A] [answered]
Hello, I created an account two days ago. I was waiting for it to be approved but did not get any notification. When I went to the app to log in or check status it was giving me the option to sign in but gave me an error even though I was using correct email & password. Now If I try and make a new account, I'm getting an error that says, "Registration Closed Server Disallow registration.

**Comment by ineffyble** [comment, marked as answer]
@HollywoodPizza sounds like the instance you tried to join is having some capacity issues. Try finding a new one at https://joinmastodon.org/ that's open and sounds right for you.

**Reply by HollywoodPizza** [reply]
Thank you for the response. It's really frustating because I am a content creator so what server do you suggest I use considering I can't use this one?

**Reply by ineffyble** [reply]
Which one did you try to sign up for?

**Reply by HollywoodPizza** [reply]
peoplemaking.games

**Reply by ineffyble** [reply]
Please read the server rules, description, etc, before deciding, but a few options I found:

https://mastodon.gamedev.place/about
https://gamemaking.social/about
https://gamedev.lgbt/about

**Reply by HollywoodPizza** [reply]
thanks a ton really appreciate you!
